<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DecodeWidjet</class>
 <widget class="QWidget" name="DecodeWidjet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="encode_seq_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Путь к файлу с закодированной последовательностью</string>
   </property>
  </widget>
  <widget class="QPushButton" name="decode_button">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>610</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Декодировать</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="alphabet_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="alphabet_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Пусть к файлу с алфавитом</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="encode_seq_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="encode_seq_file_dialog_button">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>210</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Обзор</string>
   </property>
  </widget>
  <widget class="QLabel" name="decode_label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Декодирование</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="alphabet_file_dialog_button">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Обзор</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
